packages/@react-spectrum/datepicker/stories/DateRangePicker.stories.tsx

```diff
@@ -176,6 +185,10 @@ storiesOf('Date and Time/DateRangePicker/styling', module)
     'errorMessage',
     () => render({errorMessage: 'Dates must be after today', validationState: 'invalid'})
   )
+  .add(
+    'invalid with time',
+    () => render({validationState: 'invalid', granularity: 'minute', defaultValue: {start: toZoned(parseDate('2020-02-03'), 'America/New_York'), end: toZoned(parseDate('2020-02-12'), 'America/Los_Angeles')}})
+  )
```

Comment:
Should the time fields also be invalid? At the moment they are not which could perhaps give the wrong impression to a user, especially when the cause of an invalid date range is the times set rather than the day itself. Same question for the DatePicker as well

Reply:
Maybe yeah. I thought it looked pretty over the top to have three error triangle icons at once though...